src/client.ts

```diff
@@ -833,9 +833,9 @@ export class C8Client extends Fabric {
     return search.getListOfViews();
   }
 
-  createView(viewName: string, properties?: Properties) {
+  createView(viewName: string, properties?: Properties,primarySort?:Array<PrimarySortFields>) {
```

Comment:
No need to take primary sort separately. everything falls under properties only. just update the docs with properties.

Reply:
no it is a separate field as per my understanding
```{
  "name": "string",
  "primarySort": [
    {}
  ],
  "links": {
    "[collection-name]": {
      "analyzers": [
        "string"
      ],
      "fields": {
        "field-name": [
          {}
        ]
      },
      "includeAllFields": true,
      "storeValues": "string",
      "trackListPositions": true
    }
  },
  "type": "search"
} ```

Reply:
Replace properties with link